src/main/java/com/conveyal/analysis/results/GridResultWriter.java

```diff
@@ -50,45 +50,43 @@ public class GridResultWriter extends BaseResultWriter {
     private static final long HEADER_LENGTH_BYTES = 9 * Integer.BYTES;
 
     /**
-     * The number of different travel time cutoffs being applied when computing accessibility for each origin.
+     * The number of thresholds (time or cumulative access) being applied when computing values for each origin.
      * The number of values stored per origin cell in an accessibility results grid.
```

Comment:
Should these lines be combined?
```suggestion
     * The number of values stored per origin the the results grid -- that is, the number thresholds (time or cumulative access) being applied when computing values for each origin.
```

Reply:
I took a pass at rewriting the whole comment for conciseness:


> The number of thresholds (time or cumulative access) stored in the grid and used for computing values per origin.
> 
> Note: Only the number of thresholds is stored, not their values. Proper interpretation requires threshold values from the regional analysis. Storing values would require a file format change.

Applied in https://github.com/conveyal/r5/pull/954/commits/f1a52bd27102ba218c2386e65090b53a8a582583